### PR TITLE
Cvsl 1327 dps header update for cvl

### PR DESCRIPTION
**Pull request by nishanthmahasamudram**

### CODE REVIEW

File: server/app.ts [hunk 1]
```diff
@@ -48,6 +49,7 @@ export default function createApp(services: Services): express.Application {
   }
 
   app.use(authorisationMiddleware)
+  app.get('*', getFrontendComponents(services))
```
Comment (andrewrlee) [inline server/app.ts:52]: I think this will mean we load the header and footer for the prisoner image endpoint

Comment (nishanthmahasamudram) [reply server/app.ts:52]: This is to load header and footer components from URL(https://frontend-components-dev.hmpps.service.justice.gov.uk) and assign to res.locals. So, that as soon as our app loads we can fetch and populate.

**Review by andrewrlee — APPROVED** [review]
LGTM

File: server/app.ts (hunk 1 above)
Comment (nishanthmahasamudram) [reply server/app.ts:52]: Raised a ticket to tidy up this https://dsdmoj.atlassian.net/browse/CVSL-1374

**Pull request merged**